BuildGenerated.sh

```diff
@@ -102,6 +105,31 @@ if [ -z ${SKIPDOWNLOAD+x} ]; then
   $PYTHON2 -u get_discovery_documents.py --destination_dir $DISCOVERY_DOC_DIR
   # Patch discovery docs
   dotnet run --project $TOOLS_DIR/DiscoveryDocPatcher/DiscoveryDocPatcher.csproj -- $DISCOVERY_DOC_DIR
```

Comment:
I think patching should happen after revision only changes reverted. Else, the copy of the original discovery doc that patching leaves behind will have a different revision number than the one used for generation. Not the end of the world, but still...

There's no problem in trying to patch an already patched discovery doc, the patcher is prepared for that and does nothing.

Reply:
Discussed offline - it's fine as-is.